blueocean-dashboard/src/main/js/components/PipelinePage.jsx

```diff
@@ -21,12 +21,17 @@ const RestPaths = Paths.rest;
 const classicConfigLink = (pipeline) => {
     let link = null;
     if (Security.permit(pipeline).configure()) {
-        link = <a href={buildClassicConfigUrl(pipeline)} target="_blank"><Icon size={24} icon="ActionSettings" style={{ verticalAlign: 'baseline' }} /></a>;
+        link = (
+            <a href={buildClassicConfigUrl(pipeline)} title={webTranslate('toast.configure', { defaultValue: 'Configure' })} target="_blank">
+                <Icon size={24} icon="ActionSettings" style={{ verticalAlign: 'baseline' }} />
+            </a>
+        );
     }
     return link;
 };
 
 const translate = i18nTranslator('blueocean-dashboard');
+const webTranslate = i18nTranslator('blueocean-web');
```

Comment:
this is confusing - it has to get i18n from another module?

Reply:
I saw this used in the RunDetails.jsx (https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-dashboard/src/main/js/components/RunDetails.jsx#L28) and I thought this translation is related more to the ones that already are in web, I can move it in dashboard if you think it would make more sense there

Reply:
Yeah it’s prbably fine